setup.py

```diff
@@ -71,5 +71,5 @@ def find_meta(meta):
     ],
     python_requires=">=3.6.*",
     tests_require=["tox", "virtualenv", "wheel"],
-    entry_points={"console_scripts": ["pysemver = semver:main"]},
+    entry_points={"console_scripts": ["pysemver = semver.cli:main"]},
```

Comment:
Not sure, but should we distinguish the old script from the new one? Maybe we should add a "3" suffix at the name:

```suggestion
    entry_points={"console_scripts": ["pysemver3 = semver.cli:main"]},
```

Reply:
I guess eventually we can symlink `pysemver` to `pysemver3`.

Reply:
Would that be outside of the scope of `setup.py`? I was just wondering if it is really something we should need to take care of.

I assume, this is only needed when you have semver2 and semver3 both installed on the system and want to distinguish the scripts. In that case, it is probably needed. On the other side, as semver2 is not supported anymore, I think it's a minor issue.

Maybe leave it as it is (as `pysemver`) it's a minor issue. People will complain if they need it. :smile:

Reply:
I think people might complain if we rename to `pysemver3` 😉